spec/support/utils/profile-testing-util.js

```diff
@@ -0,0 +1,30 @@
+const enums = require("../../../lib/main/constants/enumConstants.js");
+const { generateRandomString,} = require("./general-testing-util.js");
+const generateRandomProfile = (username, createdAt, age, bmi, height, weight, shouldParse = true) => {
+const generateRandomDate = () => {
+     const randomTimestamp = Math.random() * Date.now();
+     return new Date(randomTimestamp).toISOString().split('T')[0];
+    };
+
+     module.exports = {
+     generateRandomDate
+    };
+     username = generateRandomString(20);
+     age = Math.floor(Math.random() * (100 - 12 + 1)) + 12; // Random age between 18 and 100
+     bmi = Math.random() * (30 - 18) + 18; // Random BMI between 18 and 30
```

Comment:
There are already functions that do this dont repeat. 

look at generate random whole number